client/my-sites/hosting/test/index.js

```diff
@@ -345,26 +331,4 @@ describe( 'Hosting Configuration', () => {
 			} );
 		} );
 	} );
-
-	describe( 'Staging site', () => {
```

Comment:
@arthur791004 would be great to include this test in https://github.com/Automattic/dotcom-forge/issues/7513.

Reply:
Added by https://github.com/Automattic/wp-calypso/pull/91276/commits/64352d2289b01039482ec3bcc5fae80f9e229f32